src/earn/EarnEnterAmount.tsx

```diff
@@ -110,14 +133,15 @@ function EarnEnterAmount({ route }: Props) {
   const onTokenPickerSelect = () => {
     tokenBottomSheetRef.current?.snapToIndex(0)
     AppAnalytics.track(SendEvents.token_dropdown_opened, {
-      currentTokenId: token.tokenId,
-      currentTokenAddress: token.address,
-      currentNetworkId: token.networkId,
+      currentTokenId: transactionToken.tokenId,
+      currentTokenAddress: transactionToken.address,
+      currentNetworkId: transactionToken.networkId,
```

Comment:
should this be inputToken? since that's the token that the user selects

Reply:
Changed in 1da7fac29afd55d61d0717cdc66e28568f4cdca2.